components/crud-web-apps/common/frontend/kubeflow-common-lib/projects/kubeflow/src/public-api.ts

```diff
@@ -69,3 +69,14 @@ export * from './lib/services/date-time.service';
 
 export * from './lib/panel/panel.module';
 export * from './lib/panel/panel.component';
+
+export * from './lib/namespace-select/namespace-select.component';
+export * from './lib/form/name-namespace-inputs/name-namespace-inputs.component';
+export * from './lib/form/name-namespace-inputs/name-input/name-input.component';
+export * from './lib/form/positive-number-input/positive-number-input.component';
+export * from './lib/form/advanced-options/advanced-options.component';
+export * from './lib/details-list/details-list-item/details-list-item.component';
+export * from './lib/form/submit-bar/submit-bar.component';
+export * from './lib/form/step-info/step-info.component';
+export * from './lib/form/submit-bar/submit-bar.component';
+export * from './lib/form/step-info/step-info.component';
```

Comment:
Initially, those exports did make any sense since we don't use directly these classes. But then I saw that the Angular compiler throws an error if we remove them. From v9 the compiler throws an error if any component is exported in `NgModule` and not included in a `public_api.ts`. So my suggestion is that you could better explain your changes in the commit messages. It will be very helpful for the reviewers.

Reply:
I started the upgrade work months ago so I don't fully remember what errors precisely would I encountered that required a fix. Any manual changes are either due to an error occurring or because the upgrade guide said this change should be done. Luckily there are only a few manual changes that were needed.

Reply:
> I started the upgrade work months ago so I don't fully remember what errors precisely would I encountered

The need for good commit messages is to tackle exactly this problem. We use commit messages to come back to commits and be able to reason about the changes that happened in that commit.

Let's keep it in mind to have better commit messages in the future

Reply:
I completely agree. My focus for these commit titles and descriptions was mainly for reproducibility so I forgot to add descriptions of why certain changes were needed.